pytest_timeout.py

```diff
@@ -347,15 +384,23 @@ def _parse_marker(marker):
         raise TypeError("Multiple values for method argument of timeout marker")
     elif len(marker.args) >= 2:
         method = marker.args[1]
-    if len(marker.args) > 2:
+    if len(marker.args) >= 3 and disable_debugger_detection is not NOTSET:
+        raise TypeError(
+            "Multiple values for disable_debugger_detection argument of timeout marker"
+        )
+    elif len(marker.args) >= 3:
+        disable_debugger_detection = marker.args[2]
```

Comment:
In any case I don't think we should allow this as a positional argument, only as keyword if at all.  I'm a bit surprised that we allow method as positional, but I guess this comes from a time before keyword-only arguments were a thing in Python.

Reply:
Dropping marker support per the comment above.